### DISCUSSION TITLE
wss protocol issue and sockets pending state

### DISCUSSION THREAD

**Discussion by siva1002** [miguelgrinberg/python-socketio#1193] [General]
Hi ,
    Ec2 instance is not reponds quick with socket request some request went to pending state.socket send request to 40-50 requests to the server is there any possibility to opitize the request

**Comment by miguelgrinberg** [comment]
It is really hard to understand what is the issue here. A request in pending state is not indicative of a problem, if you are using the long polling transport it is perfectly normal for requests to be kept pending for several seconds.
